### PR TITLE
Ruowan/gen api scenario

### PR DESCRIPTION
**Pull request by ruowan**

### CODE REVIEW

File: lib/commands/generate-static-api-scenario.ts [hunk 1]
```diff
@@ -19,31 +19,43 @@ export const builder: yargs.CommandBuilder = {
...
-  const readmeMd: string = pathResolve(argv.readme);
-
-  const autorestConfig = await getAutorestConfig(argv, readmeMd);
-  const fileRoot = dirname(readmeMd);
-  const swaggerFilePaths: string[] = autorestConfig["input-file"].map((it: string) =>
-    pathResolve(fileRoot, it)
-  );
+  const swaggerFilePaths: string[] = (argv.swaggers || []).map((it: string) => pathResolve(it));
+  let tag = "default";
+  if (argv.readme !== undefined) {
```
Comment (raych1) [inline lib/commands/generate-static-api-scenario.ts:38]: I assume it supports scenario file generation when both of swaggers parameter and readme parameter are passed in, right?

File: lib/apiScenario/gen/staticTestScenarioGenerator.ts [hunk 2]
```diff
@@ -209,10 +209,11 @@ export class StaticApiScenarioGenerator {
     for (const it of this.exampleDependencies) {
       resourceTypes.add(it.fullResourceType);
     }
+    if (this.opts.rules?.includes("operations-list")) {
+      console.log("generate operations list");
+      await this.generateListOperationTestScenario();
```
Comment (raych1) [inline lib/apiScenario/gen/staticTestScenarioGenerator.ts:214]: Do we change generation behavior with moving this call outside of resourceTypes loop?

Comment (ruowan) [reply lib/apiScenario/gen/staticTestScenarioGenerator.ts:214]: No

File: lib/commands/generate-static-api-scenario.ts (hunk 1 above)
Comment (ruowan) [reply lib/commands/generate-static-api-scenario.ts:38]: Yes.

**Review by raych1 — APPROVED** [review]

**Pull request merged**